src/components/search/search.css

```diff
@@ -1,30 +1,106 @@
+* {
+  font-family: "Noto Sans";
+  font-style: normal;
+  font-weight: 900;
+}
+
+.search-background {
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  position: fixed;
+  z-index: 1;
+  width: 100%;
+  height: 100vh;
+  background-color: rgba(0, 0, 0, 0.1);
+  transition: all 0.3s ease-in-out;
+}
 
-.search-background{
-    bottom:0;
-    position:fixed;
-    z-index: 1;
-    width: 100vw;
-    height: 100vh;
-    background-color: rgba(0, 0, 0, 0.1);
+.search-content {
+  border-radius: 16px 16px 0px 0px;
+  bottom: 0;
+  width: 393px;
+  height: 772px;
+  position: fixed;
+  z-index: 1;
+  background: #fff4ec;
 }
 
-.search-content{
-    border-radius: 16px 16px 0px 0px;
-    bottom: 0;
-    width: 393px;
-    height: 772px;
-    position: fixed;
-    z-index: 1;
-    background: #FFF4EC;
+.search-title {
+  margin-top: 47px;
+  margin-left: 24px;
 }
 
-.search-title{
-    margin-top: 47px;
-    margin-left: 24px;
+.search-title > h2 {
+  font-size: 40px;
+  line-height: 54px;
+  color: #c98686;
 }
 
-.search-title>h2{
-    font-size: 40px;
-    line-height: 54px;
-    color: #C98686;
-}
\ No newline at end of file
+.hide-search {
+  transform: translateY(100%);
+  opacity: 0;
+}
+
+.form-div {
+  display: flex;
+  width: 100%;
+}
+form {
```

Comment:
not using a more specific selector means that all future forms that will be added on the page will have the same styling. Even though this project may not have more than one form, its a better approach to customise each one by adding a custom class.